File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.java
class Solution {
    public int[] searchRange(int[] nums, int target) {
        int firstOccurrence = -1;
        int secondOccurrence = -1;
        
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == target) {
                if (firstOccurrence == -1) {
                    firstOccurrence = i;
                }
                secondOccurrence = i;
            }
        }
        
        return new int[] { firstOccurrence, secondOccurrence };
    }
}

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        ans = [-1, -1]
        for i in range(len(nums)):
            if target == nums[i] and ans[0] == -1:
                ans[0] = i
                ans[1] = i
            elif target == nums[i]:
                ans[1] = i
        return ans